lib/data-types.js

```diff
@@ -82,9 +82,8 @@ ABSTRACT.prototype.stringify = function stringify(value, options) {
 /**
  * A variable length string. Default length 255
  *
- * Available properties: `BINARY`
- *
- * @property STRING
+ * @property BINARY
```

Comment:
I find `@property` a bit misleading. According to JSDoc-docs, all we need is `@memberof`
http://usejsdoc.org/tags-namespace.html